mruby/src/value/mod.rs

```diff
@@ -75,6 +80,32 @@ impl Value {
     }
 }
 
+impl Clone for Value {
+    fn clone(&self) -> Self {
+        let ruby_type = self.ruby_type();
+        if ruby_type == Data {
```

Comment:
since `ruby_type` is an enum, I think it is more idiomatic to have this if condition be:

```rust
if let Ruby::Data = self.ruby_type() {
```